### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by pablobm**
Fixes https://github.com/thoughtbot/administrate/issues/2000

Any thoughts on this?

### CODE REVIEW

**Comment by nickcharlton** [comment]
This looks good!

Shall we remove the line from the `CONTRIBUTING.md` file? What do you think about making this show up in the Docs, too?

**Comment by pablobm** [comment]
Good points. Done now.

**Review by nickcharlton — APPROVED** [review]

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+
+Only the the latest version of Administrate is supported at a given time. If you find
```
Comment (ruanltbg) [inline SECURITY.md:5]: there is a double `the` here.

Comment (ruanltbg) [reply SECURITY.md:5]: cc @pablobm

Comment (pablobm) [reply SECURITY.md:5]: PRs welcome :-)

**Pull request merged**